packages/flutter/lib/src/widgets/form.dart

```diff
@@ -193,6 +193,16 @@ class FormState extends State<Form> {
     }
     return !hasError;
   }
+
+  /// Checks every [FormField] that is a descendant of this [Form] if `hasError` is true,
+  /// and returns the list of invalid fields, or an empty list if there are none.
+  ///
+  /// The form will not be rebuilt.
+  List<FormFieldState<dynamic>> get invalidFields {
```

Comment:
Would this be better as an `Iterable` so that it's lazy, for those users that don't need a full List?